x-pack/plugin/eql/src/test/java/org/elasticsearch/xpack/eql/planner/QueryFolderOkTests.java

```diff
@@ -121,7 +121,7 @@ public void test() {
         PhysicalPlan p = plan(query);
         assertEquals(EsQueryExec.class, p.getClass());
         EsQueryExec eqe = (EsQueryExec) p;
-        assertEquals(1, eqe.output().size());
+        assertEquals(0, eqe.output().size());
```

Comment:
Why 0 now?

Reply:
Because no field is needed for event queries. The projection made the assumption the timestamp is needed, which is not the case.